docs/standard/io/how-to-read-text-from-a-file.md

```diff
@@ -26,17 +26,17 @@ The following example shows a synchronous read operation within a console app. T
 > [!IMPORTANT]
 > The example assumes that a file named *TestFile.txt* already exists in the same folder as the app.  
 
- [!code-csharp[Conceptual.BasicIO.TextFiles#3](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source3.cs#3)]
- [!code-vb[Conceptual.BasicIO.TextFiles#3](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source3.vb#3)]  
+:::code language="csharp" source="./snippets/how-to-read-text-from-a-file/csharp/sync-console/Program.cs" id="":::
+:::code language="vb" source="./snippets/how-to-read-text-from-a-file/vb/sync-console/Program.vb" id="":::
```

Comment:
```suggestion
:::code language="csharp" source="snippets/how-to-read-text-from-a-file/csharp/sync-console/Program.cs":::
:::code language="vb" source="snippets/how-to-read-text-from-a-file/vb/sync-console/Program.vb":::
```

The `id` attribute should be omitted when not set, and the leading `./` in the path is redundant. Also, if we're moving files - I would try to follow *vb* and *cs* for directory names, it's a bit more concise - and the shorthand *cs* helps with long file path concerns.

Reply:
@IEvangelist The current guidelines for folder structure uses vb, csharp, and fsharp.

I think using cs instead of csharp requires changing the guidelines first, then update all the existing csharp folder names. This might need to be discussed separately, as it affects multiple existing folders, not only folders in this PR.

Reply:
Hi @Youssef1313, where are these guidelines you speak of - I'm actually not familiar with them. I do know that many folders that exist today do follow the naming convention that you called out, however; I believe we could do better. Just because something was done one way doesn't mean its the best way forward. I'm not proposing any sweeping changes now, I'm talking about things that are new.

Reply:
The folder structure convention is stated [here](https://docs.microsoft.com/en-us/contribute/dotnet/dotnet-contribute#example-folder-structure). (They doesn't render well currently, I opened https://github.com/MicrosoftDocs/Contribute/pull/417 to fix that, you may look at the PR for better readability). If I remember correctly, @adegeo pointed me to this folder structure to use it when I move samples to the corresponding docs folder.